src/main/java/duke/Events.java

```diff
@@ -0,0 +1,104 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+/**
+ * Events is a sub class of Task, because it allows the user to key in
+ * additional data: the start date and the end date of the event task. By
+ * default, all events are initially marked as undone. The user can either
+ * choose to key in dates in MMM dd YYYY format, or normal Strings that
+ * represents time for them (e.g. Sunday.) Since immutability is employed,
+ * when a user toggles the event task as done or undone, a new event task
+ * will be returned, but with the exact same description, from and to
+ * timeframe.
+ *  @author Muhammad Reyaaz
+ *  @version %I% %G%
+ *  @see Task
+ *  @since 11
+ */
+class Events extends Task {
```

Comment:
I suggest changing the class name to Event (singular)
```suggestion
class Event extends Task {
```